### ISSUE TITLE
Termine mit Einladungen für alle Projektteilnehmer sichtbar

### ISSUE THREAD

**Issue by gharlan** [yakamara/redaxo_prozer#163]
Zurzeit sind Termine mit Einladungen nur für die Eingeladenen sichtbar. Ich fände es sinnvoller, wenn alle Projektteilnehmer diesen Termin sehen können.

**Comment by gharlan** [comment]
Es fällt bei uns immer wieder auf, dass das bei unserer Verwendung sehr sinnvoll wäre.

@Christian-H Ist es bei euch weiterhin explizit so gewollt, dass Termine mit Einladungen für andere (nicht eingeladene) Projektteilnehmer gar nicht sichtbar sind?

**Comment by Christian-H** [comment]
Ja, bei uns ist weiterhin explizit gewollt, dass Termine mit Einladungen für andere nicht sichtbar sind. Da wir 18-22 Personen sind, die mit dem Prozer arbeiten, wäre unser Kalender zugemüllt, wenn wir alle Termine sehen würden. Insbesondere der von unserem Geschäftsführer. 
Das ist auch für Prozer sinnvoll, da das erklärte Ziel ist, dass Prozer für eine Nutzerzahl bis zu 50 Personen ausgelegt ist.

**Comment by staabm** [comment]
@Christian-H  Alle Projektteilnehmer  != Alle Mitarbeiter. Oder sind bei Euch jegliche Mitarbeiter in allen Projekten?

**Comment by Christian-H** [comment]
Richtig: Alle Projektteilnehmer != Alle Mitarbeiter
Trotzdem sind im laufe eines Jahres viele Mitarbeiter in vielen Projekten und der Gf halt auch. Alleine ich selbst bin in 62 Projekten drin. Ein Mitarbeiter meistens mind. 20 wenn er anfängt und 30 bis 40 ist vielleicht der durchschnitt. 
Ja, wir brauchen es so wie es ist.
